src/Hedgehog/Script.fsx

```diff
@@ -57,8 +58,8 @@ Gen.printSample genLeapYear
 
 // Fails due to integer overflow
 Property.print <| property {
-    let! x = Gen.int
-    let! y = Gen.int
+    let! x = Gen.int <| Range.constantBounded ()
+    let! y = Gen.int <| Range.constantBounded ()
```

Comment:
Later, we may also provide some convenience 'default' combinators in `Gen` so that it's still possible to write, say `Gen.int`, without having to pass a `Range`. Something like `Gen.Default.int`.

Hypothetically, these convenience 'default' combinators will use some bounded range internally. This might actually be a good idea also for the Haskell version (happy to add them in both places).